docs/tips-and-tricks.md

```diff
@@ -113,6 +113,64 @@ docker create \
 
 
 ## Get the news headlines in Homer
+
+### Mapping Fields
```

Comment:
Great doc ! Could you add a link in `docs/configuration.md` (l121) to this section?